bottery/cli.py

```diff
@@ -30,7 +31,11 @@ def cli():
 @cli.command('startproject')
 @click.argument('name')
 def startproject(name):
-    # Must validate projects name before creating its folder
+    # name validation - only lowercase letters and underscores
+    if not re.match("^[a-z_]*$", name):
```

Comment:
You could use `str.isidentifier()` instead of a regex operation: https://docs.python.org/3.3/library/stdtypes.html#str.isidentifier

Easier to maintain and gets the same result :)

Reply:
`str.isidentifier()` allows numbers in the string, which doesn't fit in with Django project naming convention. Do I still go with it?

Reply:
@asi7296 actually, Django uses `str.isidentifier()` to validate project's name.